impl/client/src/main/javascript/web/dojo/pentaho/common/Dialog.js

```diff
@@ -174,6 +176,17 @@ define(["dojo/_base/declare", "dijit/_WidgetBase", "dijit/_TemplatedMixin", "dij
               this.popup.set('title',this.title);
             },
 
+            setupResponsiveness: function () {
```

Comment:
Please, name the method as `_setupResponsiveness` to indicate its protected nature.